src/sig_stfl/lms/sig_stfl_lms.h

```diff
@@ -14,12 +14,6 @@ OQS_API OQS_STATUS OQS_SIG_STFL_alg_lms_sha256_h5_w1_keypair(uint8_t *public_key
 
 OQS_SIG_STFL_SECRET_KEY *OQS_SECRET_KEY_LMS_SHA256_H5_W1_new(void);
 
-/* Convert LMS secret key object to byte string */
```

Comment:
Can you the description command back to the new serialize and deserialize function?

Reply:
These got moved with the description from the header file.